aif360/sklearn/metrics/metrics.py

```diff
@@ -119,42 +119,34 @@ def ratio(func, y, *args, prot_attr=None, priv_group=1, sample_weight=None,
     return numerator / denominator
 
 
-# =========================== SCORER FACTORIES =================================
-def make_difference_scorer(diff_func):
-    """Make a scorer from a 'difference' metric (e.g.
+# =========================== SCORER FACTORY =================================
+def make_scorer(score_func, ratio, **kwargs):
```

Comment:
Can we change `ratio` to `is_ratio` and set the default to False? Difference metrics are more common.